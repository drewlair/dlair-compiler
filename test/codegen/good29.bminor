abs: function integer(n: integer) = {
	if (n < 0) return -1*n;
    else return n;
}

digits: array [6] integer = {
	14,
	8,
	6168,
	42,
	0,
	74546
};

main:function integer () = {
	digits[4] = -468;
	i: integer = 0;
	for (; abs(digits[i]) == digits[i];) {
		print digits[i];
		i = i + 1;
		print ' ';
	}
	print '\n';

	return 10;
}