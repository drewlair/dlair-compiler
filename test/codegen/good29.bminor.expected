14 8 6168 42 
